src/libmeasurement_kit/cmdline/dns_injection.cpp

```diff
@@ -19,10 +20,10 @@ int main(const char *, int argc, char **argv) {
     mk::ooni::DnsInjection test;
     int ch;
 
-    while ((ch = getopt(argc, argv, "b:nv")) != -1) {
+    while ((ch = mkp_getopt(argc, argv, "b:nv")) != -1) {
```

Comment:
For all the portable standard library symbols, we always use the `mkp_foo` version. A set of macros in `"../portable/api.h"` would determine whether we use our own implementation (the one starting with `mkp_`) or the one provided by the system. In the latter case, macros are used to turn `mkp_foo` into `foo`.